content/docs/manual/access-control.md

```diff
@@ -0,0 +1,201 @@
+
+*Access control* enables Zenoh instances to filter (allow or deny) messages,
+depending on certain characteristics of individual messages and their respective source or destination.
+*Authentication* on the other hand allows Zenoh instances to identify certain characteristics in other instances they connect to,
+which are used to match the instances with configured subjects in the ACL policies and apply the rules accordingly on the exhanged messages.
+
+The configuration of access control policies is done via a [configuration file](../configuration).
+
+---------
+## ACL configuration
+
+ACL configuration mainly consists of three components: `rules`, `subjects` and `policies`, to which is added the `default_permission` (`allow` or `deny`) to be applied on messages that do not match the configured policies.
+
+The `enabled` boolean field allows to enable and disable ACL. Note that ACL config cannot be updated at runtime, and requires a restart of the instance to reflect the changes.
+
+Below is the example config we will be analyzing in this documentation.
+
+```json
+{
+  access_control: {
+    "enabled": true,
+    "default_permission": "allow",
+  
+    "rules": [
+      {
+        "id": "deny pub/sub",
+        "permission": "deny",
+        "flows": ["ingress", "egress"],
+        "messages": [
+          "declare_subscriber",
+          "put",
+          "delete",
+        ],
+        "key_exprs": [
+          "demo/example/**",
+        ],
+      }
+    ],
+  
+    "subjects": [
+      {
+        "id": "example subject",
+        "interfaces": [
+          "lo0",
+          "en0",
+        ],
+        "cert_common_names": [
+          "example.zenoh.io"
+        ],
+        "usernames": [
+          "zenoh-example1",
+          "zenoh-example2",
+        ],
+      }
+    ],
+  
+    "policies": [
+      {
+        "rules": [
+          "deny pub/sub"
+        ],
+        "subjects": [
+          "example subject"
+        ],
+      },
+    ]
+  }
+}
+```
+
+### rules
+
+Each rule within the `rules` list is identified by a unique `id` string. Rules apply on matched messages based on their individual characteristics:
+
+- `messages`: types of messages to apply the rule on. Supported message types are the following:
+  - Declare Subscriber (`declare_subscriber`)
+  - Publication (`put`)
+  - Delete (`delete`)
+  - Declare Queryable (`declare_queryable`)
+  - Query (`query`)
+  - Reply (`reply`)
+- `flows`: applies rule on incoming messages (`ingress`), outgoing messages (`egress`), or both directions. If this field is not provided in the config, the rule will apply to both directions by default.
+- `key_exprs`: the rule applies on messages for which the key matches one of the given key expressions. For more details on key expression matching, see [Key Expressions](http://localhost:1313/docs/manual/abstractions/#key-expression).
```

Comment:
localhost link
Did you mean this:  https://zenoh.io/docs/manual/abstractions/#key-expression

Reply:
Oops 😅